mautic/mautic
# Add company_id search command

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
In some cases search from company list based on company name does not work for us. We add new company based on company_id



#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Create contacts and assign them to company
3. Go to company list
4. See if click on number of contacts works like before 




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10702"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
